src/utils/localRequestHelper.ts

```diff
@@ -0,0 +1,18 @@
+import type { NextPageContext } from 'next/dist/shared/lib/utils';
+import type { GetServerSidePropsContext, PreviewData } from 'next/types';
+import type { ParsedUrlQuery } from 'querystring';
+
+export const getRequestHeaders = (
```

Comment:
Shouldn't this function be called something like `getAuthHeaders` instead of `getRequestHeaders`?